core/meta/inc/TClass.h

```diff
@@ -309,12 +311,14 @@ friend class TProtoClass;
    TClass(const TClass& tc) = delete;
    TClass& operator=(const TClass&) = delete;
 
+   void LoadClassInfo() const;
+
 protected:
-   TVirtualStreamerInfo     *FindStreamerInfo(TObjArray* arr, UInt_t checksum) const;
-   void                      GetMissingDictionariesForBaseClasses(TCollection& result, TCollection& visited, bool recurse);
-   void                      GetMissingDictionariesForMembers(TCollection& result, TCollection& visited, bool recurse);
-   void                      GetMissingDictionariesWithRecursionCheck(TCollection& result, TCollection& visited, bool recurse);
-   void                      GetMissingDictionariesForPairElements(TCollection& result, TCollection& visited, bool recurse);
+   TVirtualStreamerInfo *FindStreamerInfo(TObjArray* arr, UInt_t checksum) const;
```

Comment:
Please keep the original layout (function name aligned); clang-format is currently incapable to do what we really want in this area.